src/cli.rs

```diff
@@ -51,14 +51,17 @@ impl Cli {
   /// Access the directory name.
   #[inline]
```

Comment:
Why are all these functions `#[inline]` by the way? They don't seem performance critical and might also be short enough to be inlined by LLVM anyway

Reply:
That is… interesting. I can see your reasoning. I personally would rather trust the compiler (with lto) to figure out my hot paths and what to inline there though 🤷‍♂️

Reply:
@killercup I believe the compiler will _never_ inline functions that cross crate boundaries. It'll only ever auto-inline _within_ the same crate. So to allow for optimizing, it requires the `#[inline]` attribute - and then it can decide whether or not to inline.

Note that this is different from `#[inline(always)]`, as we don't want to force the compiler to inline. We just want to hint at the compiler that this function might be worth inlining.